src/main/java/org/dimdev/riftloader/RiftLoader.java

```diff
@@ -29,7 +29,9 @@
     public static final RiftLoader instance = new RiftLoader();
     private static final Logger log = LogManager.getLogger("RiftLoader");
 
-    public final File modsDir = new File(Launch.minecraftHome, "mods");
+    public File modsDir = new File(Launch.minecraftHome, "mods");
+    public final File modsVersionSpecifiedDir = new File(Launch.minecraftHome, "mods/rift/@MCVERSION@");
```

Comment:
```suggestion
    public final File modsVersionSpecifiedDir = new File(modsRiftDir, "@MCVERSION@");
```
And put this after `modsRiftDir` to resolve errors.